frontend/src/__tests__/cypress/cypress/support/commands/odh.ts

```diff
@@ -57,6 +58,16 @@ declare global {
         response?: OdhResponse,
       ): Cypress.Chainable<null>;
 
+      interceptOdh(
+        type: 'GET /api/groups-config',
+        response?: OdhResponse<GroupsConfig>,
```

Comment:
I think I made a mistake with the accelerator endpoints using `response?`. I believe responses should be required. Can you also update other 3.
```suggestion
        response: OdhResponse<GroupsConfig>,
```